### DISCUSSION TITLE
Is it possible to completely skip .gzip generation

### DISCUSSION THREAD

**Discussion by Bartuz** [ElMassimo/vite_ruby#265] [Q&A] [answered]
In my workflow compressed assets are not needed as external service handles that for us.

I can't see an option to disable compression completely. Is it possible? I hope to save a bit of time by skipping compression 🤔

**Comment by ElMassimo** [comment, marked as answer]
Hi @Bartuz, `gzip` files are not generated, the logs simply display the file size if compression was applied.

You can disable this report by using [`reportCompressedSize`](https://vitejs.dev/config/build-options.html#build-reportcompressedsize):

```ts
defineConfig({
  build: {
    reportCompressedSize: false,
  },
})
```

Reporting is fast enough that in most cases it's not worth disabling, but if you have large files then it makes sense.
